tests/rustdoc-ui/edition-2024-doctest-error-output.stdout

```diff
@@ -0,0 +1,32 @@
+
+running 1 test
+test tests/rustdoc-ui/edition-2024-doctest-error-output.rs - (line 10) ... FAILED
+
+failures:
+
+---- tests/rustdoc-ui/edition-2024-doctest-error-output.rs - (line 10) stdout ----
+
+thread 'main' panicked at /tmp/rustdoctestSZq9aS/doctest_bundle_2024.rs:6:1:
```

Comment:
> rustdoctestSZq9aS

This needs normalizing

Reply:
Arf, missed that one, thanks!

Reply:
Ah actually I wrongly normalized. Move the test, should be all good now.